packages/pages/src/common/src/function/internal/loader.test.ts

```diff
@@ -26,27 +28,35 @@ describe("loadTemplateModules", () => {
 
   it("loads and transpiles raw templates", async () => {
     const functionFile: path.ParsedPath[] = [
-      path.parse("tests/fixtures/src/functions/http/[param].ts"),
+      path.parse(path.join(httpPath, "[param].ts")),
     ];
 
     const functionModules = await loadFunctionModules(
       functionFile,
       true,
       projectStructure
     );
-    commonTests(functionModules, "param-47543");
+    if (path.sep === path.posix.sep) {
+      commonTests(functionModules, "param-47543");
+    } else {
+      commonTests(functionModules, "param-19926");
+    }
   });
 
   it("loads transpiled templates", async () => {
     const functionFile: path.ParsedPath[] = [
-      path.parse(path.resolve("tests/fixtures/src/functions/http/[param].js")),
+      path.parse(path.resolve(path.join(httpPath, "[param].js"))),
```

Comment:
I don't think you need path.join if you're using path.resolve

> All methods on path should work cross-platform, and use the platform-specific separator.

https://github.com/nodejs/node/issues/12156#issuecomment-290912558